solver/src/liquidity.rs

```diff
@@ -117,6 +117,14 @@ pub struct ConstantProductOrder {
     pub settlement_handling: Arc<dyn SettlementHandling<Self>>,
 }
 
+impl PartialEq for ConstantProductOrder {
```

Comment:
Does this mean a SushiSwap pool and a Uniswap v2 pool on the same token pair with same reserve values should be considered equal?

Reply:
I am going to remove this and compare field directly for the one unit test we needed this.